docs/requirements.txt

```diff
@@ -7,4 +7,5 @@ jupyter-book
 sphinx_external_toc
 sphinxcontrib-mermaid
 astroid~=2.15 # https://github.com/readthedocs/sphinx-autoapi/issues/407
-sphinx-autoapi
\ No newline at end of file
+sphinx-autoapi
+nirtorch
```

Comment:
This will make NIR depend on pytorch (bc nirtorch depends on torch), is this necessary? For example, if I'm using a jax-based library, I'd prefer not to have to import torch as well.

Reply:
Perhaps we could have this as a dev_requirement only?

Reply:
now I see that it's only the docs/requirements, so all good!